docs/guides/user-containers/readme.md

```diff
@@ -262,16 +262,106 @@ allowed per workflow.
     GFS2 filesystems have special considerations since the mount directory contains directories for
     every compute node. See [GFS2 Index Mounts](#gfs2-index-mounts) for more info.
 
+### Targeting Nodes
+
+For container directives, compute nodes must be assigned to the workflow. The NNF software will
+trace the compute nodes back to their local NNF nodes and the containers will be executed on those
+NNF nodes. The act of assigning compute nodes to your container workflow instructs the NNF software
+to select the NNF nodes that run the containers.
+
+For the `jobdw` directive that is included above, the servers (i.e. NNF nodes) must also be assigned
+along with the computes.
+
```

Comment:
If a compute node not associated with any of the Rabbits assigned in the Servers object are specified, what happens?

Reply:
Good question. The NNF file system wouldn't be available on the rabbit and therefore the mount directory passed into the container won't exist. I'll look into this more and add any fixes, if necessary.

Reply:
Maybe the webhook should check this?